### PR TITLE
Redesing profile page

### PR DESCRIPTION
**Pull request by coolbii**
# Revise Profile layout from last time

[Profile UI](https://github.com/Rhodanthe1116/lyrics-typing/issues/23)

## Changes made

- change TrackId into track's info(track name, artist name)

## Things to work on

- tracks' cover haven't been replace yet
- some layout may not be good enough

### CODE REVIEW

File: components/TypingInput.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ const TypeInput: FC<{
 
       // Save record.
       const newResult: TypingResult = {
-        wpm: Math.round(((60 / duration) * correctChar) / 5),
+        wpm: Math.round(((60 / duration) * correctChar)),
```
Comment (Rhodanthe1116) [inline components/TypingInput.tsx:124]: wpm是wpm
cpm是cpm

其實我在想record的schema可以不用存wpm或cpm

File: pages/profile.tsx [hunk 2]
```diff
@@ -110,21 +138,53 @@ const ProfilePage = () => {
...
+        <div className="md:mb-8 mb-4">
+         
           {/* <p>{user.id}</p> */}
-          <div className="mt-2 flex">
-            <div className="flex-1 text-center">
-              <h2 className="text-4xl">{user?.typingRecords?.length || 0}</h2>
-              <p>Total Completed </p>
+          <div className="flex">
+            <div className="flex-1 ">
+              <p className="md:text-3xl font-semibold text-left pl-6">Average CMP</p>
```
Comment (Rhodanthe1116) [inline pages/profile.tsx]: CPM

File: components/TypingInput.tsx (hunk 1 above)
Comment (coolbii) [reply components/TypingInput.tsx:124]: 那這樣CPM要從哪個table調資料

**Comment by Rhodanthe1116** [comment]
雖然還可以在雕一下，但我需要所以先merge

**Pull request merged**